Resources/Private/Translations/en/Main.xlf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file original="" product-name="Neos.GoogleAnalytics" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="accounts" xml:space="preserve">
+                <source>Authenticated accounts</source>
+            </trans-unit>
+            <trans-unit id="site.name" xml:space="preserve">
+                <source>Site</source>
+            </trans-unit>
+            <trans-unit id="site.gaProfile" xml:space="preserve">
+                <source>Google Analytics Profile</source>
```

Comment:
Maybe add a screenshot that shows where to find the profile id. In the German GA UI it is not very clear what the profile actually is.